submissions/Anas-Abuhamed/assignment8.js

```diff
@@ -0,0 +1,35 @@
+// Task 1
+const editor = {
+    title: "my first blog",
+     getUpperTitle() {
+        return this.title.toUpperCase();
+     }
+}
+const getTitle = editor.getUpperTitle.bind(editor);
+console.log(getTitle());
+// Task 2 
+const formHandler = {
+    value: "initial",
+    onChange(newValue) {
+        this.value = newValue;
+    },
+};
+function simulateInputChange(callback) { 
+    callback("updated")
+}
+simulateInputChange(formHandler.onChange.bind(formHandler)) // for this solution, bind create a new copy of the function and replace this with formhandler
```

Comment:
Please use arrow function as another solution here and explain why it could work